.gitignore

```diff
@@ -1,23 +1,82 @@
-# Compiled class file
-*.class
+# Bot Config
+config.yml
 
-# Log file
-*.log
+# IntelliJ
+out/
+*.ipr
+*.iml
+modules.xml
+/.idea
```

Comment:
`.idea` is excluded, but `.idea/icon.svg` and ` .idea/saveactions_settings.xml` were still committed while not having an explicit include. Is this intentional?